### PR TITLE
Feature/cloud data

### PR DESCRIPTION
**Pull request by arc-arnob**

### CODE REVIEW

File: src/data/fetch_dataset.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    with zipfile.ZipFile(zip_file, "r") as f:
+        f.extractall(extract_to)
+
+# Download and extract testing data
+# test_zip_path, _ = urllib.request.urlretrieve(test_URL)
+extract_files(test_zip_path, EXTRACT_DIR)
+
+# Download and extract training data
+# train_zip_path, _ = urllib.request.urlretrieve(train_URL)
+train_zip_path = './data/raw/train.zip'
```
Comment (emilssipols11) [inline src/data/fetch_dataset.py]: you have this defined twice

File: src/data/fetch_dataset.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# Test Link: https://drive.google.com/uc?export=download&id=1I5RSGrXX7qFAEcFouAIirhICF4S3cLF9
+# Train Link: https://drive.google.com/uc?export=download&id=1N2vhSR7Zi2qYbtxK4-unO8dvzo8oji7z
+# Val Link: https://drive.google.com/file/d/1-EoxS7YPMXC3iYZF46GPH4BNOeWnZJmf/view?usp=drive_link
+
+
+import urllib.request
+import zipfile
+import os
+
+# TODO: Fix Train download, too big for autoscan. Swap link and you'll see things getting downloaded
```
Comment (emilssipols11) [inline src/data/fetch_dataset.py]: this one you can remove

File: src/data/fetch_dataset.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+# Create the directory if it doesn't exist
+if not os.path.exists(EXTRACT_DIR):
+    os.makedirs(EXTRACT_DIR)
+
+def extract_files(zip_file, extract_to):
+    with zipfile.ZipFile(zip_file, "r") as f:
+        f.extractall(extract_to)
+
+# Download and extract testing data
+# test_zip_path, _ = urllib.request.urlretrieve(test_URL)
```
Comment (emilssipols11) [inline src/data/fetch_dataset.py]: also these are not necessary anymore

**Review by RadoT29 — APPROVED** [review]
Looks good, nice addition!

**Pull request merged**